frontend/text-editor/README.md

```diff
@@ -37,7 +37,7 @@ This command is going to search for the file located in `frontend/src/app/main/u
 
 ## How it works?
 
-The text editor divides the content in three elements: `root`, `paragraph` and `inline`. An `inline` in terms of content is a styled element that it is displayed in a line inside a block and an `inline` only can have one child (a Text node). A `paragraph` is a **block** element that can contain multiple `inline`s (**inline** elements).
+The text editor divides the content in three elements: `root`, `paragraph` and `textSpan`. An `textSpan` in terms of content is a styled element that it is displayed in a line inside a block and an `textSpan` only can have one child (a Text node). A `paragraph` is a **block** element that can contain multiple `textSpan`s (**textSpan** elements).
```

Comment:
Algunas sugerencias de mejora de legibilidad:

The text editor divides the content into three elements: `root`, `paragraph` and `textSpan`. In terms of content a `textSpan` is a styled element displayed on a line within a block. A `textSpan` can only have one child (a Text node). A `paragraph` is a **block** element that can contain multiple `textSpan`s (**textSpan** elements).